.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,145 @@
+
+* **Breaking changes** are removals of features such as props, changes that affect the [box model](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Box_Model/Introduction_to_the_CSS_box_model), or dramatic changes in branding or appearance
+* **Minor changes** are new features, animations, props, or visual options
+* **Patches** are defect fixes that do not remove features, alter pixel dimensions related to the box model, nor add new features. If an intended feature was not working in a previous release, changing that feature to match the original design counts as a patch even if it affects the box model
+
+### Code style and conventions
+
+Here are some dos and don'ts to consider when writing code.
+
+#### React
+
+* DO make components self-contained. A component should not know anything about other components, except for its direct children
+* DO provide a clear, prop-based API to the component. Avoid allowing consumers to customize styles by passing in `className` or `style` as this is not a clear API. Use a prop with known values instead
+* DO use React component state for ephemeral UI state within components, while avoiding redux or other application state containers
+* DO use dependencies when needed, such as lodash functions or other open source React components
+* DO use tds-core components and other tds-community components judiciously
+* DO make components compatible with React 15 or greater
+* DO use [ponyfills](https://github.com/sindresorhus/ponyfill) when using native JavaScript APIs with low browser support. The alternative is to require that any consumer of your component include a global polyfill in their app, making your component less self-contained, thought this may be preferable in some cases
+* DO forward `rest` props onto the primary HTML element of the component so that consumers can still attach global HTML attributes such as `id`.
+* DON'T hardcode content without giving consumers the ability to override it with other copy or languages
+
+#### Styling (CSS Modules, Sass)
+
+* DO scope component class names
+* DON'T specify external margins. Components must be able to fit into various layouts
+* DON'T use HTML elements or IDs as CSS selectors. Only use class names
+* DON'T hardcode pixel values. Use tds-core components or relative values such as rem instead
+
+#### Utility modules
+
+* DO add them to the `/shared` directory with its own `package.json` file
+* DO configure `private: true` within package.json
+
+### Samples
+
+If this is your first time contributing to TDS Community, you can see some [sample
+components](https://github.com/telusdigital/tds-community/tree/master/samples) as inspiration.
+
+## Making pull requests
+
+When changes to the codebase are ready for review by a member of the DPA, you may submit a pull request. Once a pull request
+is approved, a DPA member may merge your branch to `master` and changes will be automatically deployed.
```

Comment:
> "changes will be automatically deployed."

Not yet....... we won't have automated pipeline publishing set up yet. Its still manual. Maybe leave this part out and that is something for us to talk about and figure out how to handle this in the interim?